docs/source/tutorials/otx_cli/how_to_train/detection.rst

```diff
@@ -86,17 +91,18 @@ This dataset contains images of blood cells. It's a great example to start with,
 
   {'data': 
     {'train': 
-      {'ann-files': '/home/<username>/training_extensions/BCCD/train/_annotations.coco.json',
-       'data-roots': '/home/<username>/training_extensions/datasets/BCCD/train'},
+      {'ann-files': '/home/<username>/training_extensions/data/wgisd/coco_annotations/train_bbox_instances.json',
+       'data-roots': '/home/<username>/training_extensions/data/wgisd/data'},
     'val':
-      {'ann-files': '/home/<username>/training_extensions/datasets/BCCD/valid/_annotations.coco.json',
-       'data-roots': '/home/<username>/training_extensions/datasets/BCCD/valid'},
+      {'ann-files': '/home/<username>/training_extensions/data/wgisd/coco_annotations/test_bbox_instances.json',
+       'data-roots': '/home/<username>/training_extensions/data/wgisd/data'},
     'test':
-      {'ann-files': '/home/<username>/training_extensions/datasets/BCCD/valid/_annotations.coco.json',
-       'data-roots': '/home/<username>/training_extensions/datasets/BCCD/valid'}
+      {'ann-files': '/home/<username>/training_extensions/data/wgisd/coco_annotations/train_bbox_instances.json',
+       'data-roots': '/home/<username>/training_extensions/data/wgisd/data'}
     }
   }
 
+``Ann-files`` contains a path to the annotation., while ``data-roots`` is a path to the folder, where images are stored.
```

Comment:
``Ann-files`` contains a path to the annotation**s,** while ``data-roots`` is a path to the folder, where images are stored.